src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+class Block{
+	public static int blockNumber = 123;
+	public Block(int blockNumber){
+		Block.blockNumber = 123;
+	}
+	public  int getNumber(){
+		return blockNumber;
+	}
+}
+class Street{
```

Comment:
None of your major new classes have header comments. Please add the class header comments.